packages/media-sound/audacious-plugins/audacious-plugins.exlib

```diff
@@ -15,37 +15,38 @@ Note: some plugins are not on by default and have to be enabled in Preferences -
 
 LICENCES="
     BSD-2 [[ note = [ alsa audpl compressor crossfade ladspa mixer mpg123 mpris2 oss4 resample sdlout unix-io ] ]]
-    BSD-3 [[ note = [ psf xsf ] ]]
-    GPL-2 [[ note = [ aac alarm amidi-plug aosd blue_scope bs2b cue filewriter hotkey jack m3u metronom mms neon null OSS pulse_audio sid sndfile sndstretch tongen vtx xspf ] ]]
+    BSD-3 [[ note = [ opus pipewire psf xsf ] ]]
+    GPL-2 [[ note = [ aac amidi-plug aosd blue_scope bs2b cue filewriter hotkey jack m3u metronom mms neon null OSS pulse_audio qhotkey sid sndfile sndstretch tongen vtx xspf ] ]]
     GPL-3 [[ note = [ asx cdaudio-ng cd-menu-items ffmpeg flacng gtkui notify pls search-tool skins song_change ] ]]
     LGPL-2 [[ note = [ vorbis ] ]]
     LGPL-2.1 [[ note = [ adplug console ] ]]
-    MIT [[ note = [ albumart cairo-spectrum crystalizer lyricwiki voice_removal ] ]]
+    MIT [[ note = [ albumart bitcrusher cairo-spectrum crystalizer lyricwiki voice_removal ] ]]
     public-domain [[ note = [ aac ] ]]
 "
 SLOT="0"
 
 MYOPTIONS="
     (
         alsa [[ description = [ Support ALSA audio output and MIDI synthesizing with AlsaMidi. ] ]]
-        pulseaudio [[ description = [ Support PulseAudio output. ] ]]
-        oss [[ description = [ Support OSSv4 audio output. ] ]]
         jack [[ description = [ Support JACK audio output. ] ]]
+        oss [[ description = [ Support OSSv4 audio output. ] ]]
+        pipewire [[ description = [ Support Pipewire output. ] ]]
+        pulseaudio [[ description = [ Support PulseAudio output. ] ]]
         qtaudio [[ description = [ Support QtMultimedia output. ] requires = [ qt5 ] ]]
     ) [[ number-selected = at-least-one ]]
 
     aac [[ description = [ Support AAC decoding. ] ]]
-    fluidsynth [[
-        description = [ Support MIDI synthesizing with fluidsynth. ]
-        requires = [ alsa ]
-    ]]
     bs2b [[ description = [ Improve headphone listening of stereo audio records via libbs2b. ] ]]
     cd [[ description = [ AudioCD support. ] ]]
     cue [[ description = [ Support for CUE metadata files from CD/DVDs. ] ]]
     dbus [[ description = [ Control audacious via dbus mpris interface. ] ]]
     ffmpeg [[ description = [ Support decoding tons of audio codecs and access formats via ffmpeg. ] ]]
     flac [[ description = [ Support FLAC decoding and dumping audio out to a flac file. ] ]]
-    gtk [[ description = [ Build the GTK+ 2 GUI. ] ]]
+    fluidsynth [[
+        description = [ Support MIDI synthesizing with fluidsynth. ]
+        requires = [ alsa ]
+    ]]
+    gtk [[ description = [ Build the GTK+ GUI. ] ]]
```

Comment:
Have you removed the "2" from the description on purpose? According to 
https://github.com/audacious-media-player/audacious-plugins/commit/6c628b44203ab6eddab55b52aff0d24f4ad6e07d
gtk3 looks supported now as well, however in the exlib below gtk support still requests 2 by specifying `x11-libs/gtk+:2`.